src/helpers/deviceSecurity.ts

```diff
@@ -0,0 +1,10 @@
+import JailMonkey from "jail-monkey";
+
+export const isDeviceJailbroken = (): boolean => {
+  try {
+    return JailMonkey.isJailBroken();
+  } catch (error) {
+    // Fail closed - treat errors as jailbroken device
+    return true;
```

Comment:
are we confident on defaulting it to `true` here?

if anything happens with the lib which ends up resulting in an error then suddenly all users will be locked out of the app. Or if let's say some devices doesn't support it, then some users will be locked out and it may take some time until we realize this is happening and deploy a fix